lib/orbital/models/response.rb

```diff
@@ -79,6 +79,18 @@ def self.orbital_response_params(response)
           :params_tx_ref_num => extract(response, 'tx_ref_num'),
         }
       end
+
+      def first_reference_id
+        params_tx_ref_num
+      end
+
+      def second_reference_id
+        params_order_id
+      end
+
+      def gateway_error_code
+        params_resp_code
```

Comment:
👍 

I'll merge it for now, but I think we should get to the bottom of this as it would simplify our work regarding [Payment retries](https://github.com/killbill/killbill-payment-retries-plugin). Note that we currently apply the [same rules](https://github.com/killbill/killbill-payment-retries-plugin/blob/master/src/main/java/org/killbill/billing/plugin/payment/retries/rules/RulesComputer.java#L38-L39) between CyberSource and Orbital (since we get the raw Orbital response code from CyberSource already).